ui-components/src/commonMain/kotlin/org/jetbrains/kotlinconf/ui/Gallery.kt

```diff
@@ -44,7 +45,6 @@ fun GalleryApp() {
                 SwitcherPreview()
                 TogglePreview()
                 TopMenuButtonPreview()
-                TopMenuTitlePreview()
```

Comment:
Removed this component, as it was super simple, and only used as part of `MainHeader`. We can probably do this for a couple more similar components later.